interface-definitions/service_ipoe-server.xml.in

```diff
@@ -174,10 +174,31 @@
                   </leafNode>
                 </children>
               </node>
+              <leafNode name="lua-username">
+                <properties>
+                  <help>Username function</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>Name of the function in the Lua file to construct usernames with</description>
+                  </valueHelp>
```

Comment:
Better have a constraint on the function name by not allowing weird characters/umlauts.

```suggestion
                  </valueHelp>
                  <constraint>
                      #include <include/constraint/alpha-numeric-hyphen-underscore.xml.i>
                  </constraint>
```